README.md

```diff
@@ -146,101 +159,112 @@ Following are different directives currently available.
   * [Copy Column](docs/directives/copy.md)
   * [Merge Columns](docs/directives/merge.md)
   * [Keep Columns](docs/directives/keep.md)
-  * [Swap Column](docs/directives/swap.md)
+  * [Swap Columns](docs/directives/swap.md)
   * [Split To Columns](docs/directives/split-to-columns.md)
   * [Fill Null or Empty Columns](docs/directives/fill-null-or-empty.md)
   * [Replace Column Names - Bulk](docs/directives/columns-replace.md)
-  * [Sanatizes Column Names](docs/directives/cleanse-column-names.md)
+  * [Sanatize Column Names](docs/directives/cleanse-column-names.md)
   * [Set Column Names](docs/directives/set-columns.md)
 * **NLP**
   * [Stemming Tokenized Words](docs/directives/stemming.md)
 * **Functions**
     * [JSON](docs/functions/JSON.md)
     * [Types](docs/functions/Types.md)
-  
+
+
 ## Performance
 
-Initial performance tests shows that with medium set of directives for transforming data, wrangler is able to process at ~ 60K records/second. The rates below are specified as '**records/second**'. More details and test results will be available [here](docs/performance.md)
+Initial performance tests shows that with a set of directives of medium complexity for
+transforming data, *Wrangler* is able to process at about 60K records/second. The rates
+below are specified as *records/second*. Additional details and test results are available
+[here](docs/performance.md).
 
 | Directive Complexity | Column Count| Records | Size | Mean Rate | 1 Minute Rate | 5 Minute Rate | 15 Minute Rate |
 |----------------------|-------------|---------|------|-----------|---------------|---------------|----------------|
-| Medium | 18 | 13,499,973 | 4,499,534,313 | 64,998.50 | 64,921.29 | 46,866.70 | 36,149.86 | 
-| Medium | 18 | 80,999,838 | 26,997,205,878 | 62,465.93 | 62,706.39 | 60,755.41 | 56,673.32 | 
+| Medium | 18 | 13,499,973 | 4,499,534,313 | 64,998.50 | 64,921.29 | 46,866.70 | 36,149.86 |
+| Medium | 18 | 80,999,838 | 26,997,205,878 | 62,465.93 | 62,706.39 | 60,755.41 | 56,673.32 |
+
 
 ## Wrangler Service
 
-Wrangler is integrated as a CDAP Service to support REST based interactive ways to wrangle data. The main objective of having this service is to make it easy for interactively generating directives required for parsing data. This service does not support full scale big data processing.
+Wrangler is integrated as a CDAP Service to support HTTP RESTful-based interactive
+wrangling of data. The main objective of this service is to make it simple and easy to
+interactively generate the directives required for parsing a given data set. This service
+does not support full-scale big data processing.
```

Comment:
Both the service and the transform use the same common library for applying directives to data: wrangler-core.jar. The service is just a tool for users to interactively apply directives to sample data and see what it does to the data. Since the directives are interpreted by a common library, when the user executes those same directives in the transform plugin on millions of rows (we usually call this "operationalizing" them), those directives should be executed the exact same way. So, you are correct in saying the "service does not support full scale big data processing", but the library that is common to both the service and the transform supports it.